.changeset/wild-cougars-juggle.md

```diff
@@ -0,0 +1,7 @@
+---
+"reframed": patch
+---
+
+Update `writable-dom` to fix empty inline script tag issue
```

Comment:
moving forward I wonder if we want to also keep the conventional commits format for changesets too (that's what's being done in workers-sdk)

not a huge deal for this PR anyways 🙂